rustls/src/client/client_conn.rs

```diff
@@ -1,3 +1,12 @@
+use alloc::sync::Arc;
+use alloc::vec::Vec;
+use core::fmt;
```

Comment:
Group these into `core::{fmt, mem, io}` while you're at it?

Reply:
I left the import tidying out at request of ctz.